Description: Use local copy of docbookx.dtd
 http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd is added to debian
 dir and used instead of the online one so the package can build with no
 network access.
Author: Scott Kitterman <scott@kitterman.com>
Origin: vendor
Forwarded: not-needed

Index: libopendbx-1.4.6/utils/doc/opendbx-utils.en.xml
===================================================================
--- libopendbx-1.4.6.orig/utils/doc/opendbx-utils.en.xml	2012-11-03 22:31:11.226725868 -0400
+++ libopendbx-1.4.6/utils/doc/opendbx-utils.en.xml	2012-11-03 22:31:15.022725927 -0400
@@ -1,5 +1,5 @@
 <!DOCTYPE set PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
-          "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+          "../../debian/xml/docbookx.dtd">
 
 <set>
 	<title>OpenDBX</title>
Index: libopendbx-1.4.6/doc/opendbx.en.xml
===================================================================
--- libopendbx-1.4.6.orig/doc/opendbx.en.xml	2012-11-03 22:29:36.550724394 -0400
+++ libopendbx-1.4.6/doc/opendbx.en.xml	2012-11-03 22:34:20.582728815 -0400
@@ -1,5 +1,5 @@
 <!DOCTYPE set PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
-          "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+          "../debian/xml/docbookx.dtd">
 
 <set>
 	<title>OpenDBX</title>
